src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -224,6 +233,8 @@ public bool NotUsingHttps(string url)
                     return;
                 }
 
+                await RecreateIndexes(databaseRecord);
```

Comment:
what if this is not viable for this node?

Reply:
Not viable how?

Reply:
I can create this on a different node, no?